ofono/bridge.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import os
+from time import sleep
+
+import pydbus
+from dbus.mainloop.glib import DBusGMainLoop
+from gi.repository import GLib
```

Comment:
Same for `gi` module

Reply:
It appears the gi module is easily installable once I do `apt install python-gi`. So, it appears, we really do need the "zerophone-essentials" virtual package =)

Reply:
Package created (`zerophone-requirements`)